rero_ils/permissions.py

```diff
@@ -219,3 +219,58 @@ def decorated_view(*args, **kwargs):
             else:
                 abort(403)
     return decorated_view
+
+
+def can_read_update_delete_acquisition_factory(record, *args, **kwargs):
```

Comment:
is this can be a generic: can_read_update_delete_library_factory, no?

Reply:
Same comment for the next functions.

Reply:
It may be interesting to keep this if we add a new role for acquisitions